user_claims_test.go

```diff
@@ -314,3 +314,56 @@ func TestUserAccountIDValidation(t *testing.T) {
 		t.Fatal("expected validation issues")
 	}
 }
+
+func TestSourceNetworkValidation(t *testing.T) {
+	ukp := createUserNKey(t)
+	uc := NewUserClaims(publicKey(ukp, t))
+
+	uc.Limits.Src = "192.0.2.0/24"
+	vr := CreateValidationResults()
+	uc.Validate(vr)
+
+	if !vr.IsEmpty() {
+		t.Error("limits should be valid")
+	}
+
+	uc.Limits.Src = "192.0.2.1/1"
+	vr = CreateValidationResults()
+	uc.Validate(vr)
+
+	if !vr.IsEmpty() {
+		t.Error("limits should be valid")
+	}
+
+	uc.Limits.Src = "192.0.2.0/24,2001:db8:a0b:12f0::1/32"
```

Comment:
Maybe add in a test with different spaces or tabs around the comma separations.

Reply:
Added support for whitespace and updated test in user limits